### ISSUE TITLE
switch off gamification

### ISSUE THREAD

**Issue by Nimeggis** [MEITREX/frontend#1]
From gits-frontend created by [LG1401](https://github.com/LG1401): IT-REX-Platform/gits-frontend#116  

## Description
A lecturer or student should be able to switch the gamification features for a course (for all course participants if lecturer).

## Acceptance Criteria
- [ ] gamification can be switched of

**Comment by Nimeggis** [comment]
@LG1401 this would need some kind of backend integration first I guess? To store if it’s of or on

**Comment by LG1401** [comment]
@Nimeggis yes, such a backend integration would be needed first. It has not been done in the previous project
